chapters/extensions/cleanup.md

```diff
@@ -0,0 +1,118 @@
+# Cleanup Extensions
+
+> These are extensions that are unofficially called "cleanup extension". The Vulkan Guide defines them as cleanup extensions due to their nature of only adding a small bit of functionality or being very simple, self-explanatory extensions in terms of their purpose.
+
+# VK_KHR_driver_properties
+
+> Promoted to core in Vulkan 1.2
+
+This extension adds more information to query about each implementation. The [VkDriverId](https://www.khronos.org/registry/vulkan/specs/1.2-extensions/html/vkspec.html#VkDriverId) will be a registered vendor's ID for the implementation. The [VkConformanceVersion](https://www.khronos.org/registry/vulkan/specs/1.2-extensions/html/vkspec.html#VkConformanceVersion) displays which version of [the Vulkan Conformance Test Suite](../chapters/vulkan_cts.md) the implementation passed.
+
+# VK_EXT_host_query_reset
+
+> Promoted to core in Vulkan 1.2
+
+This extension allows an application to call `vkResetQueryPool` from the host instead of needing to setup logic to submit `vkCmdResetQueryPool` since this is mainly just a quick write to memory for most implementations and little overhead to calling.
```

Comment:
nit: drop "and little overhead to calling"? (because grammar is odd, and what it says is implied by "quick write" anyway)